config/standalone-config.properties

```diff
@@ -23,22 +23,22 @@
 #singlenode.restServerPort=9091
 
 # Cert file for TLS communication.
-#singlenode.certFile=../config/cert.pem
+singlenode.certFile=../config/cert.pem
 
 # Key file for TLS communication
-#singlenode.keyFile=../config/key.pem
+singlenode.keyFile=../config/key.pem
 
 # Password file for default password handler
-#singlenode.passwdFile=../config/passwd
+singlenode.passwdFile=../config/passwd
 
 #Username for default password handler implementation
-#singlenode.userName=admin
+singlenode.userName=admin
 
 #Password associated with the default user
-#singlenode.passwd=1111_aaaa
+singlenode.passwd=1111_aaaa
 
 # Enable TLS for segmentstore and controller.
-#singlenode.enableTls=true
+singlenode.enableTls=true
 
 # Enable authentication/authorization.
-#singlenode.enableAuth=true
+singlenode.enableAuth=true
```

Comment:
I don't think TLS or auth should be enabled by default

Reply:
Fixed.